common/templates/ellipsis.html

```diff
@@ -26,7 +26,8 @@
         <button ng-if="edit" type="button" class="edit-action-button chaise-btn chaise-btn-tertiary chaise-btn-link icon-btn" ng-click="edit()" target="_blank" tooltip-placement="bottom" uib-tooltip="Edit">
             <span class="chaise-btn-icon glyphicon glyphicon-pencil"></span>
         </button>
-        <button ng-if="unlink" type="button" class="delete-action-button chaise-btn chaise-btn-tertiary chaise-btn-link icon-btn" ng-click="unlink()" tooltip-placement="bottom" uib-tooltip="Unlink">
+        <!-- TODO: should this be "config.deletable && unlink && associationRef.canDelete" || "unlink" -->
```

Comment:
I think we're defining `scope.unlink` if these conditions that you mentioned here are met. Right?

Reply:
I meant to review this statement and check if that's what it was doing.